Update node and packages

ol0lll opened the pull request.

mtodor commented on sharpeye.tasks.js:
```diff
@@ -143,130 +152,150 @@ module.exports = [
+    { $: '//*[@id="edit-meta-changed"]/text()', value: ' 01/01/2018 - 00:00' }
+  ], actions: [
+    { fill: [
+      { $: 'input.select2-search__field', fill: "abc" }
```
Here we should have `value` instead of `fill`. API changes.

mtodor commented:
I have reviewed current status and here are my findings.

1. Screenshot: `admin_config_system_site-information-system_information-chrome_1280.png` - there is no URL anymore.
2. Screenshot: `admin_content_media-media-chrome_1280.png` - table cells for updated are gone.
3. Some screenshots are same. For example: `core_install_php-install_page-firefox_1280.png`, why we have diff there? Also this one: `media_3_edit-media_type_twitter_edit_form-firefox_1280.png`.
4. Here is one problematic: `node_7_edit-entity_browser_gallery-firefox_1280.png` - loading of entity browser is not finished. :(
5. Can we consider having more images with strict height? Would that solve problems like on screenshot `node_7_edit-open_sidebar_elements-firefox_1280.png`? Or on screenshot `node_add_article-modals_in_paragraphs-chrome_1280.png`?

ol0lll commented:
> 1. Screenshot: `admin_config_system_site-information-system_information-chrome_1280.png` - there is no URL anymore.
The `hide` command in task definition did not work correctly, should be hidden since url could be different on some systems, see

> 2. Screenshot: `admin_content_media-media-chrome_1280.png` - table cells for updated are gone.
The 'hide' command works differently now, switched the task to 'replace', where xpath selection of inner text is possible.

> 3. Some screenshots are same. For example: `core_install_php-install_page-firefox_1280.png`, why we have diff there? Also this one: `media_3_edit-media_type_twitter_edit_form-firefox_1280.png`.
I reset them using `git checkout origin/8.x-3.x <filename>`.

> 4. Here is one problematic: `node_7_edit-entity_browser_gallery-firefox_1280.png` - loading of entity browser is not finished. :(
Aligned task to wait for form in iframe to be displayed

> 5. Can we consider having more images with strict height? Would that solve problems like on screenshot `node_7_edit-open_sidebar_elements-firefox_1280.png`? Or on screenshot `node_add_article-modals_in_paragraphs-chrome_1280.png`?
Added viewport settings for `node_7_edit-open_sidebar_elements-firefox_1280.png`.
Setting height for task checking an element only is not possible atm, so I opted for a fullPage screenshot for `node_add_article-modals_in_paragraphs-chrome_1280.png`.
